src/web/app/pages-student/student-home-page/student-home-page.component.spec.ts

```diff
@@ -507,11 +519,16 @@ describe('StudentHomePageComponent', () => {
           isSubmitted: true,
         },
       ],
+      isFeedbackSessionsLoading: false,
+      hasFeedbackSessionsLoadingFailed: false,
+      isTabExpanded: true,
+      hasPopulated: true,
     };
 
     component.courses = [studentCourse];
     component.isCoursesLoading = false;
-    fixture.detectChanges();
+    // fixture.detectChanges();
+    fixture.autoDetectChanges();
```

Comment:
I'm not too familiar with testing in Angular, what would this change be for? Also can remove the commented out code if you don't need it

Reply:
My bad! I forgot to change it back after playing around with it :)